src/components/dashboard/GapByRace.vue

```diff
@@ -29,12 +77,14 @@
 import { computed } from "vue";
 
 import GapChart from "@/components/dashboard/GapChart.vue";
+import KeyPerformanceIndicator from "./KeyPerformanceIndicator.vue";
```

Comment:
use the `@/components/...` style instead for consistency

Reply:
Sounds good!